mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -2839,7 +2846,13 @@ private static void AddImplementedMembers(FrameworkTypeEntry typeEntry, MemberRe
             foreach (var implementedMember in implementedMembers)
             {
                 var value = msxdocxSlashdocFormatter.GetDeclaration(implementedMember);
-                WriteElementWithText(e, "InterfaceMember", value);
+                var interfaceMemberElement = WriteElementWithText(e, "InterfaceMember", value);
+
+                interfaceMemberElement.AddFrameworkToElement(typeEntry.Framework);
+                if (typeEntry.IsMemberOnLastFramework(mi))

```

Comment:
Just like the check you added in `UpdateType` for `typeEntry.IsOnFirstFramework`, there should be a similar check here at the member level ... you can use `typeEntry.IsMemberOnFirstFramework(member)` to determine when you should clear the list of interface members (unless this is already happening somewhere and it's just not in these diffs?))

Reply:
oh yes, the logic is already there at line 2808:
```
            var fingerprint = DocUtils.GetFingerprint(mi);
            if (typeEntry.IsMemberOnFirstFramework(mi))
            {
                ClearElement(root, "Implements");
            }
```

Reply:
excellent!